src/index.rs

```diff
@@ -13,6 +13,7 @@ const OUTPOINT_TO_ORDINAL_RANGES: TableDefinition<[u8; 36], [u8]> =
   TableDefinition::new("OUTPOINT_TO_ORDINAL_RANGES");
 const OUTPOINT_TO_TXID: TableDefinition<[u8; 36], [u8; 32]> =
   TableDefinition::new("OUTPOINT_TO_TXID");
+const OUTPUTS_TRAVERSED: TableDefinition<str, u64> = TableDefinition::new("OUTPUTS_TRAVERSED");
```

Comment:
Let's make it a general table for statistics, so we can easily use it for other things:

```suggestion
const STATISTICS: TableDefinition<u64, u64> = TableDefinition::new("STATISTICS");
```

We can create an enum for statistics:

```rust
#[repr(u64)]
enum Statistics {
  TotalOutputs = 0,
}
```

Enums with `repr(T)` can be converted to `T` with `as`: `Statistics::TotalOutputs as u64`, so we can use a `Statistics` value as a key.

Reply:
nice, that's very handy